packages/webapp/components/layouts/AccountLayout/Security/index.tsx

```diff
@@ -176,8 +204,13 @@ function AccountSecurityDefault({
             inputId="new_password"
             label="Password"
             name="password"
+            indicateStrengthAutomatically={false}
```

Comment:
Why do we choose to turn this off here?

Reply:
The design team asked that we should not show any indicator until the user interacted with it. Well, I think this should not be needed anymore after we have decided to start checking the strength once the field has value.

Reply:
Ah yes perfect, so we can remove this whole indicateauto stuff that would make it way more readable.